CMakeLists.txt

```diff
@@ -261,6 +261,14 @@ if("${HIP_FOR_NVIDIA}")
 	endif()
 endif()
 
+# Ginkgo
+# workaround for Ginkgo+CUDA install
+find_package(CUDAToolkit REQUIRED)
+target_link_libraries(DDC INTERFACE CUDA::cublas)
```

Comment:
I think it needs to be inside a condition on whether CUDA has been enabled on Kokkos

Reply:
No, the point is we need it to compile on Persee when CUDA is OFF. It is a workaround related to the fact that Ginkgo in Spack is compiled with CUDA

Reply:
Yes you are right, I can deploy an omp only version. That way if we want to test without CUDA, we activate this new environment